cobalt/browser/application.cc

```diff
@@ -687,7 +686,7 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
   // Initializes persistent settings.
   persistent_settings_ =
       std::make_unique<persistent_storage::PersistentSettings>(
-          kPersistentSettingsJson);
+          configuration::Configuration::kPersistentSettingsJson);
```

Comment:
I don't think this should be changed. You shouldn't need to create a new persistent settings json file. This will also break persistent settings for other settings. The rasterizer persistent setting is a key-value pair that lives within this json file.

Reply:
This is no being changed actually. In this PR, we need to refer this constant in multiple files, so I moved it to a different place, but it's still settings.json/